package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "torii",
-  "version": "1.0.0-beta.2",
+  "version": "1.0.0-beta.3",
```

Comment:
suggestion: the version bump should be the last commit - also, do we need to bump the version up again?